README.md

```diff
@@ -34,14 +34,17 @@ fn rdm_encipher(message: &str) -> (u16, String)
 You can try the lib using the `caesar-cli` tool:
 
 ```sh
-$ caesarlib encipher "Carolus Magnus" --offset 87
-OmDAxGE YmszGE
+$ cargo run -- --method encipher --offset 87 --text "Carolus Magnus"
+Result: lJaXUdb vJPWdb
+With Offset: 87
 
-$ caesarlib decipher "OmDAxGE YmszGE" --offset 87
-Carolus Magnus
+$ cargo run -- --method decipher --offset 87 —text "lJaXUdb vJPWdb"
+Result: Carolus Magnus
+With Offset: 87
 
-$ caesarlib encipher "Carolus Magnus" --offset 87 | caesarlib decipher --offset 87
```

Comment:
It seems that you removed the feature of reading from the stdin, which prevent us from using the pipe.
However, we can probably still do it using `xargs`